src/api/resources/project.py

```diff
@@ -58,19 +62,43 @@
 logger = logging.getLogger(__name__)
 
 
-class BaseProjectSerializer(BaseAPISerializer):
+class BaseProjectSerializer(DynamicFieldsMixin, BaseAPISerializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._cached_profiles = None
+
     countries = serializers.SerializerMethodField()
     num_sites = serializers.SerializerMethodField()
     num_active_sample_units = serializers.SerializerMethodField()
     num_sample_units = serializers.SerializerMethodField()
     tags = serializers.ListField(source="tags.all", child=TagField(), required=False)
     members = serializers.SerializerMethodField()
+    default_citation = serializers.SerializerMethodField()
+    suggested_citation = serializers.SerializerMethodField()
+    citation_retrieved_text = serializers.SerializerMethodField()
 
     class Meta:
         model = Project
         exclude = []
+        hidden_fields = ["default_citation", "user_citation", "citation_retrieved_text"]
         additional_fields = ["countries", "num_sites"]
 
+    def _get_profiles(self, obj):
+        if self._cached_profiles is None:
+            self._cached_profiles = get_profiles(obj)
+        return self._cached_profiles
+
+    def get_citation_retrieved_text(self, obj):
```

Comment:
Should change or alias the `get_citation_retrieved_text` function in the import. I see this in other files as well